### ISSUE TITLE
Color a tab

### ISSUE THREAD

**Issue by kjk** [sumatrapdfreader/sumatrapdf#2725]
It would be great to be able to color a tab (e.g. right click and choose a color). 

Why?

If you have a lot of reference PDFs open, example programming language books, it might be hard to remember which of the 10 books you have open you are currently using. If you could just say assign a color then it is easy to see which books you are currently reading.

_Originally posted by @mishaxz in https://github.com/sumatrapdfreader/sumatrapdf/discussions/2724_

Can be done with the new context menu on tabs.

**Comment by GitHubRulesOK** [comment]
Related to https://github.com/sumatrapdfreader/sumatrapdf/issues/559

**Comment by PhilBaz** [comment]
@GitHubRulesOK 

My request if further fleshed out would be to decouple the tab color from the bar, and/or add the highlight line beneath. Would this be possible? It would be a very nice upgrade.

From a user perspective the linking of colors above is not ideal. I would love, love, love either of the possibilities mocked up below.

![Image](https://github.com/user-attachments/assets/1aaf3ec5-c207-4c29-95e0-6eadf32eedc3)


ps. sorry to dupe, didn't know which was the best bet of being seen.

**Comment by PhilBaz** [comment]
Just wanted to give this a bump hoping for 3.7 integration.

**Comment by kjk** [comment]
@PhilBaz Should the color be associated with file and persist after closing / re-opening file?

**Comment by PhilBaz** [comment]
@kjk 

I think that is more related to the OPs request, which I'm piggy backing on. *** I'm sure he would like that.

My request is simply to have the ability to control the active tab ( or maybe better a highlighted underscore) as a constant theme choice. I think these requests are compatible, and I also see a lot of value in the original request, as a way to visually group documents. 

Thank you!!!

**Comment by mishaxz** [comment]
yeah the idea is just - you have a bunch of files related to x open, to y open, z, etc. and maybe right now x and y are really important to keep referencing so your color x all the same color (manually) and y another color manually.. 

by extension is not a great idea because it is common to have documents on the same topic but different formats.. e.g. pdf, epub, etc.

**Comment by mishaxz** [comment]
or you could allow for grouping like Vivaldi has Tab Stacks ,.. if you wanted to go that route, but I think colors are easier to implement. And even so, Vivaldi Tab Stacks let you specify a color for the stack and its tabs
